### PR TITLE
[Frontend] Join-leave lfg functionality

### PR DESCRIPTION
**Pull request by huscivi**
Users can join and leave the lfg.

### CODE REVIEW

File: app/frontend/src/pages/LfgPage/LfgCard.js [hunk 1]
```diff
@@ -1,16 +1,100 @@
...
+import axios from 'axios'
 
 const LfgCard = ({ group, currentUser }) =>  {
 
     const isCurrentUserCreator = currentUser && group.user.username === currentUser.username ? true : false
+    const axiosInstance = axios.create({
+        baseURL: `${process.env.REACT_APP_API_URL}`
+    })
+
+    const isUserGroupMember = (user, group) => {
```
Comment (beyzabektan) [inline app/frontend/src/pages/LfgPage/LfgCard.js:17]: You can use async here since sometimes user returns null.

**Review by beyzabektan — CHANGES_REQUESTED** [review]

File: app/frontend/src/pages/LfgPage/LfgCard.js (hunk 1 above)
Comment (huscivi) [reply app/frontend/src/pages/LfgPage/LfgCard.js]: I added null check and async function.

Comment (beyzabektan) [reply app/frontend/src/pages/LfgPage/LfgCard.js]: It seems okay now.

**Comment by beyzabektan** [comment]
Thank you for your contribution. I am merging, great job!

**Pull request merged**